tasks/opsep/update-l2pao-to-aliased-l1pao/SignFromJson.s.sol

```diff
@@ -0,0 +1,188 @@
+
+// Interface used to read various data from contracts. This is an aggregation of methods from
+// various protocol contracts for simplicity, and does not map to the full ABI of any single contract.
+interface IFetcher {
+    function overhead() external returns (uint256); // SystemConfig
+    function scalar() external returns (uint256); // SystemConfig
+    function guardian() external returns (address); // SuperchainConfig
+    function L2_BLOCK_TIME() external returns (uint256); // L2OutputOracle
+    function SUBMISSION_INTERVAL() external returns (uint256); // L2OutputOracle
+    function FINALIZATION_PERIOD_SECONDS() external returns (uint256); // L2OutputOracle
+    function startingTimestamp() external returns (uint256); // L2OutputOracle
+    function startingBlockNumber() external returns (uint256); // L2OutputOracle
+    function owner() external returns (address); // ProtocolVersions
+    function required() external returns (uint256); // ProtocolVersions
+    function recommended() external returns (uint256); // ProtocolVersions
+}
+
+interface IProxyAdmin {
+    event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
+
+    function owner() external view returns (address);
+
+    function transferOwnership(address newOwner) external;
+}
+
+contract SignFromJson is OriginalSignFromJson {
+    using LibString for string;
+
+    // Chains for this task.
+    string constant l1ChainName = "sepolia";
+    string constant l2ChainName = "op";
+
+    // Safe contract for this task.
+    GnosisSafe l2paoSafe = GnosisSafe(payable(0xb41890910b05dCba3d3dEF19B27E886C4Ab406EB));
+    address constant l2PaoSafeEOA = 0xfd1D2e729aE8eEe2E146c033bf4400fE75284301;
+
+    // Contracts we need to check, which are not in the superchain registry
+    IProxyAdmin l2pa = IProxyAdmin(0x4200000000000000000000000000000000000018);
+    // Aliased L1PAO
+    address constant aliasedL1PAO = 0x2FC3ffc903729a0f03966b917003800B145F67F3;
+    address constant unaliasedL1PAO = 0x1Eb2fFc903729a0F03966B917003800b145F56E2;
+    uint160 private constant ALIASING_OFFSET = uint160(0x1111000000000000000000000000000000001111);
+
+    // Known EOAs to exclude from safety checks.
+    address constant l2OutputOracleProposer = 0x49277EE36A024120Ee218127354c4a3591dc90A9; // cast call $L2OO "PROPOSER()(address)"
+    address constant l2OutputOracleChallenger = 0xfd1D2e729aE8eEe2E146c033bf4400fE75284301; // In registry addresses.
+    address constant systemConfigOwner = 0xfd1D2e729aE8eEe2E146c033bf4400fE75284301; // In registry addresses.
+    address constant batchSenderAddress = 0x8F23BB38F531600e5d8FDDaAEC41F13FaB46E98c; // In registry genesis-system-configs
+    address constant p2pSequencerAddress = 0x57CACBB0d30b01eb2462e5dC940c161aff3230D3; // cast call $SystemConfig "unsafeBlockSigner()(address)"
+    address constant batchInboxAddress = 0xff00000000000000000000000000000011155420; // In registry yaml.
```

Comment:
I'm guessing this is leftover from the template, I would be surprised if these show up in the state diff since they are L1 contracts. Let's delete as much dead code as we can here

Reply:
https://github.com/ethereum-optimism/superchain-ops/pull/204/commits/5055a9049e26cf7cd4c847be600d41042295e377